arrabiata/src/poseidon_3_60_0_5_5_fp.rs

```diff
@@ -0,0 +1,903 @@
+use mina_curves::pasta::Fp;
```

Comment:
dont we have this poseidon already ?
Also at some point we should move all the poseidon together

Reply:
No, we do have a poseidon with alpha = 7 and 55 rounds.
Yes, we should move all poseidon instances together one day.